client/state/inline-help/actions.js

```diff
@@ -146,22 +142,6 @@ export function hideInlineHelpPopover() {
 	};
 }
 
-export function showChecklistPrompt() {
```

Comment:
I found these actions are still used in other parts of the code:

https://github.com/Automattic/wp-calypso/blob/dc580cf2b9ae3195ece6a7381aba03165ef11baf/client/my-sites/preview/main.jsx#L25-L29

Not sure if we need to maintain the linked code or if we can remove it too as part of this PR.